workflows/metapep.nf

```diff
@@ -14,7 +14,10 @@ def checkPathParamList = [ params.input, params.multiqc_config ]
 for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
 
 // Check mandatory parameters
-if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
+if (params.input) { ch_input = file(params.input)
+} else if (!params.supported_allele_information){
+    exit 1, 'Input samplesheet not specified!'
```

Comment:
This should be changed too, I guess

Reply:
Oh yes! I wonder why this has not given me an error! I think the groovy function will be executed before that. But for completeness this should be changed!

Reply:
You mean the parameter, to a new name?

Reply:
The `exit` statement points to `Nextflow.exit(...)`, so it should be fine.